packages/waterfall-streamer/src/Utilities/dataGenerator.ts

```diff
@@ -2,24 +2,48 @@ type WaterfallDataType = 'constant' | 'random';
 let dataType: WaterfallDataType = 'constant';
 
 let counter: number = 0;
-// TODO: return the default value
-// const arraysLength: number = 4096;
-const arraysLength: number = 256;
+const arraysLength: number = 3840;
```

Comment:
rather a small detail, but this vars are all consts. I'm nore sure you really need to state their types.

Reply:
So just change this to 
```suggestion
const arraysLength = 3840;
```